src/HyperFragmentView.tsx

```diff
@@ -24,15 +24,31 @@ var HyperFragmentViewManager: any;
 
 if (Platform.OS === 'android') {
   HyperFragmentViewManager = requireNativeComponent('HyperFragmentViewManager');
+} else {
+  HyperFragmentViewManager = requireNativeComponent('HyperFragmentViewManagerIOS');
```

Comment:
@Subhash703 - why different component name for iOS? Can we have it same?

Reply:
If I change the name to `HyperFragmentViewManager ` then for some reason it is not able to find a component with name `HyperFragmentViewManager`. If I am giving any name other than `HyperFragmentViewManager` it is working fine.

Reply:
You would have to change the class name on native side as well